modules/installation-aws-permissions-iam-shared-vpc.adoc

```diff
@@ -0,0 +1,12 @@
+// Module included in the following assembly:
+// * installing/installing_aws/installing-aws-account.adoc
+
+:_content-type: REFERENCE
+[id="installation-aws-permissions-iam-shared-vpc_{context}"]
+= Trust policies for installing into a shared VPC
```

Comment:
I'm fine with moving this into the AWS VPC installation docs, but I don't want to move it into the installation configuration parameters table because I think it will be hard to find, and it will take up a lot of space in the table.